### DISCUSSION TITLE
How to access Smart Connections ChatGPT plugin

### DISCUSSION THREAD

**Discussion by pickleton89** [brianpetro/obsidian-smart-connections#264] [General]
I have tried to install the smart connections plugin in ChatGPT, but when click install, I am taken to a screen with and email field and license_key field. Is there additional information on signing in somewhere for license key? 

Thanks
Jeff

**Comment by brianpetro** [comment]
Since it's still in the experimental phase, the ChatGPT plugin requires a supporter key, which can be purchased in the plugin settings.

🌴

**Comment by oues81** [comment]
> Since it's still in the experimental phase, the ChatGPT plugin requires a supporter key, which can be purchased in the plugin settings.
> 
> 🌴



Hello,

I'm a little bit confused by the weird process for a new user and lack of clear informations. I'm searching for about 30 min to understand how to make the plugin works. A page open asking for the key but no information on how to get the license key.

Options in obsidian states that no supporter key is needed to use smart connections but cannot sync the vault...

Also, when finally understood that license is needed and the only way to generate it is by clicking the supporter button to land on default page to donate 200$ without trying anything ? I did not tried yet with openai api.

I would like to support you but considering what i just described, it doesnt help...

**Reply by brianpetro** [reply]
Hi @oues81 

I understand your concern, and I'm sorry for the confusion.

None of the features within Obsidian require a supporter key.

If your interested in the ChatGPT integrations, which do require a supporter key, I suggest you first become familiar with what the plugin has to offer within Obsidian.

To clarify my intentions, the supporter option isn't meant to be a product, but a way for people that regularly utilize Smart Connections to support the project. The extras that are offered, which currently include the experimental ChatGPT integrations, are things that either I can't ship to everyone at once, like early releases that would have too many issues for a general audience, and things that have a direct cost for me to provide, like the ChatGPT integrations which require a dedicated server, time spent managing the server, etc.

I don't spend much time promoting the supporter option because I believe it's best for me to focus my time on contributing to other aspects of the project, and also in hope that the minimal nature of the offering only attracts people that truly want to support the project rather than see "supporting" as some sort of alternative name for a service they're paying for, which would imply many more obligations that could quickly make the whole proposition no longer worthwhile from my perspective.

I hope that clarifies some of the decisions I've made. What do you think, though? Is there something you think I could do that would better help someone like yourself understand my intentions?

Thanks for your interest in supporting my work on the project,
🌴
